ote/resources/public/language/en.edn

```diff
@@ -1147,36 +1147,36 @@ You can also draw the operating area or point to the map with drawing tools. You
   :logged-out "You have been logged out."}
 
  :service-viewer
- {:open-in-geojson "Avaa tiedot GeoJSON-muodossa"
-  :operator-info "Palveluntuottajan tiedot"
-  :transport-service-info "Transport service information"
-  :not-disclosed "Ei ilmoitettu"
+ {:open-in-geojson "Open information in GeoJSON form"
+  :operator-info "Service provider information"
+  :transport-service-info "Service provider information"
```

Comment:
Should be something like "Service information"

Reply:
Good catch. Accidentally added the translation under the wrong key.